src/map/utils/charutils.cpp

```diff
@@ -2142,7 +2144,9 @@ namespace charutils
         }
         if (equipSlotID == SLOT_MAIN || equipSlotID == SLOT_RANGED || equipSlotID == SLOT_SUB)
         {
-            PChar->health.tp = 0;
+            if (equipSlotID != SLOT_RANGED || PItem == nullptr || PItem->getJobs() != 512) {  // Is not instrument
```

Comment:
Any ranged item with BRD only as the job will get through this check. Add a function to CItemArmor called like ```IsInstrument``` and have that function check the skill type. I'm sure there are other places in code that would benefit from that abstraction.